tests/layers/test_numerical_calibrator.py

```diff
@@ -839,3 +894,51 @@ def test_training():
     assert torch.allclose(
         torch.absolute(keypoints_inputs), keypoints_outputs, atol=2e-2
     )
+
+
+def test_training_learned_interior():
```

Comment:
if you decide to switch to learned vs. learned_interior make sure to update things like this here and elsewhere. Also maybe consider updating the test name to something like `test_training_learned_interior_input_keypoints`